src/js/angular/security/templates/user.html

```diff
@@ -194,16 +194,16 @@ <h3>{{'security.repo.rights' | translate}}</h3>
                                         <input class="write" type="checkbox" ng-model="grantedAuthorities.WRITE_REPO['*']" ng-checked="hasWritePermission('*')" ng-disabled="writeCheckDisabled('*')" ng-click="setGrantedAuthorities()">
                                     </td>
 								</tr>
-								<tr ng-repeat="repository in getReadableRepositories() | orderBy: 'id'">
-									<td class="repository-name">{{repository.id}}</td>
-									<td class="text-xs-center read-rights"><input class="read" type="checkbox" ng-model="grantedAuthorities.READ_REPO[repository.id]"  ng-checked="hasReadPermission(repository.id)" ng-disabled="readCheckDisabled(repository.id)" ng-click="setGrantedAuthorities()"></td>
-									<td class="text-xs-center write-rights"><input class="write" type="checkbox" ng-model="grantedAuthorities.WRITE_REPO[repository.id]" ng-checked="hasWritePermission(repository.id)" ng-disabled="writeCheckDisabled(repository.id)" ng-click="setGrantedAuthorities()"></td>
+								<tr ng-repeat="repository in getReadableRepositories() | orderBy: ['id', 'location']">
```

Comment:
Would it not be better to have the repos sorted by location first and then id, so they are kind of grouped by location?